test/core/network/protocols/websocket.test.js

```diff
@@ -132,6 +134,37 @@ describe('core/network/protocols/websocket', () => {
     });
   });
 
+  describe('upgrade connection', () => {
+    beforeEach(() => httpWs.init(entryPoint));
+
+    it('should upgrade the connection and forward and store the cookie if present when upgrading', () => {
```

Comment:
I do not understand exactly the description of this scenario,
Can you update this to be more explicit?